elements/lisk-p2p/package-lock.json

```diff
@@ -144,6 +144,43 @@
 				}
 			}
 		},
+		"@liskhq/bignum": {
+			"version": "1.3.1",
+			"resolved": "https://npm.lisk.io/@liskhq%2fbignum/-/bignum-1.3.1.tgz",
+			"integrity": "sha512-q9+NvqbpmXOqpPmV8Y+XSEIUJFMZDGyfW6rkN9Ej3nzPb/qurY/Ic2UPTeTTaj8+q/bcw5JUwTb86hi7PIziDg==",
+			"requires": {
+				"@types/node": "11.11.2"
+			},
+			"dependencies": {
+				"@types/node": {
+					"version": "11.11.2",
+					"resolved": "https://npm.lisk.io/@types%2fnode/-/node-11.11.2.tgz",
+					"integrity": "sha512-iEaHiDNkHv4Jrm9O5T37OYEUwjJesiyt6ZlhLFK0sbo4CLD0jyCOB4Pc2F9iD3MbW2397SLNxZKdDGntGaBjQQ=="
+				}
+			}
+		},
+		"@liskhq/lisk-cryptography": {
```

Comment:
I think it should not include this in the package-lock json.
can you try
`npx lerna bootstrap` in the root level?